### PR TITLE
Yautja Scimitar alternative style.

### PR DESCRIPTION
**Pull request by JustLat**
# About the pull request

Adds an alternative style for the scimitar weapon through a radial system much like the glaive and scythe.

# Explain why it's good for the game

This allows for more customization for Yautjas and allows them to pick which sprite they want to use.

# Testing Photographs and Procedure

<details>

<summary>Screenshots & Videos</summary>

https://github.com/user-attachments/assets/fe3dd142-3566-491e-8e54-625e84e8746f

Screenshot_1

Screenshot_2

Screenshot_3

</details>

# Changelog

:cl:
add: adds alt sprite style for scimitars, similar to war glaive and scythe
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Blundir — APPROVED** [review]

**Comment by Blundir** [comment]
Doesn't touch the files that i touch in reorganization, good to merge

File: code/modules/cm_preds/yaut_weapons.dm [hunk 1]
```diff
@@ -130,6 +130,17 @@
...
 	has_speed_bonus = FALSE
 
+/obj/item/weapon/wristblades/scimitar/alt
+	name = "wrist scimitar"
+	plural_name = "wrist scimitars"
+	desc = "A huge, serrated blade extending from metal gauntlets."
+	icon_state = "scim_alt"
+	item_state = "scim_alt"
+	attack_speed = 5
+	attack_verb = list("sliced", "slashed", "jabbed", "torn", "gored")
```
Comment (harryob) [inline code/modules/cm_preds/yaut_weapons.dm:140]:
```suggestion
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Pull request merged**